introduction.md

```diff
@@ -0,0 +1 @@
+The blackest of prime numbers, I participated in a Code School class and had to create an introduction. This, this is it.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.